README.md

```diff
@@ -97,6 +101,5 @@ Give a ⭐️ if you like this project!
 
 ## Acknowledgments
 
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
+- Microverse team for guidance
```

Comment:
- [ ] [OPTIONAL] Please don't forget to leave a link to your MIT.md license file 📃👍🏼